### PR TITLE
libvirt_disk: return instead of exception

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/utils_libvirt/libvirt_disk.py [hunk 1]
```diff
@@ -66,8 +67,6 @@ def get_non_root_disk_names(session):
     s, o = utils_misc.cmd_status_output(
         cmd, ignore_status=False, shell=True, verbose=True, session=session
     )
-    if s:
```
Comment (smitterl) [inline virttest/utils_libvirt/libvirt_disk.py:69]: I think you shouldn't remove these lines but do
```
if s and not ignore_status:
    raise ....
```
to maintain existing behavior.

**Review by smitterl — CHANGES_REQUESTED** [review]
@dzhengfy Thanks for this. I think it needs one small change for backward compatibility, rest LGTM.

File: virttest/utils_libvirt/libvirt_disk.py (hunk 1 above)
Comment (chloerh) [reply virttest/utils_libvirt/libvirt_disk.py:69]: +1

Comment (dzhengfy) [reply virttest/utils_libvirt/libvirt_disk.py:69]: ah... yes,  I just realize ignore_status does not take effect in  utils_misc.cmd_status_output() when session is not None.

Updated.

**Review by smitterl — APPROVED** [review]
lgtm

**Review by chloerh — APPROVED** [review]

**Pull request merged**